bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/FormUtil.java

```diff
@@ -0,0 +1,23 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~ Copyright 2024 Adobe
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+package com.adobe.cq.forms.core.components.models.form;
+
+import org.osgi.annotation.versioning.ProviderType;
+
+@ProviderType
+public interface FormUtil {
+    Boolean isEdgeDeliveryRequest();
```

Comment:
Do we want multiple implementations of this interface ?
We can directly implement the service without defining the interface.

Reply:
Also isn't there any existing util class ?

Reply:
Hi @vdua , We have component Utils but this method doesn't fit in it. This class can be used to put in any misc apis related to form which can't be accomodated in other Utils.

We don't want multiple implementations but coding on top of an interface will be a better coding precedent.

Reply:
>Do we want multiple implementations of this interface ?

This is a provider type as described in the interface, hence only one implementation can exist.

>We can directly implement the service without defining the interface.

Having an interface defines a contract, so even if there is one implementation, there is no harm having an interface

>Also isn't there any existing util class ?

NO